pom.xml

```diff
@@ -29,10 +29,9 @@
     <properties>
         <revision>3.10.0</revision>
         <changelist>-SNAPSHOT</changelist>
-        <gitHubRepo>jenkinsci/config-file-provider-plugin</gitHubRepo>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
         <maven.javadoc.skip>true</maven.javadoc.skip>
         <jenkins.version>2.303.3</jenkins.version>
```

Comment:
Note that we are not bumping the baseline here. This is to support my efforts to switch jenkinsci/bom to JENKINS-45047. This also is friendlier to users by allowing new releases to be adopted by older consumers.